config/vscode/extensions.local.json.in

```diff
@@ -0,0 +1,9 @@
+{
+	"recommendations": [
+    "formulahendry.terminal",
+		"ms-vscode.cpptools"
+	],
+	"unwantedRecommendations": [
+		"ms-vscode.cmake-tools"
```

Comment:
I had much struggles using the cmake extension as it conflicts somehow. It this disabled here if installed.